l10n_cn_sequence_voucher_manual/views/voucher_wizard_view.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="action_move_sequence_generate_wizard" model="ir.actions.act_window">
+            <field name="name">Generate Chinese Voucher Number</field>
+            <field name="res_model">account.move.sequence.generate.wizard</field>
+            <field name="view_mode">form</field>
+        </record>
+
+        <act_window name="Generate Chinese Voucher Number"
+                    res_model="account.move.sequence.generate.wizard"
+                    src_model="account.move"
+                    key2="client_action_multi"
+                    multi="True"
+                    view_mode="form"
+                    view_type="form"
+                    target="new"
+                    id="action_move_sequence_generate_wizard"/>
+
+
+        <record id="view_move_sequence_generate_wizard_form" model="ir.ui.view">
+            <field name="name">view.account.move.sequence.generate.wizard.form</field>
+            <field name="model">account.move.sequence.generate.wizard</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
+                        <group string="Generate Voucher Number Settings">
+                            <field name="prefix"/>
+                            <field name="padding"/>
+                            <field name="number_begin"/>
+                            <field name="suffix"/>
+                        </group>
+                        <group string="Has Number Vouchers" col="1">
+                            <div>vouchers below already have chinese voucher number, will rewrite it.</div>
```

Comment:
There is a boolean option 'is_rewrite' under this div,  select it will do the overwritten, otherwise does not, Vouchers below already have Chinese voucher number and will be overwritten seems a little confuse